priv/templates/phx.gen.context/schema_access_scope.ex

```diff
@@ -0,0 +1,139 @@
+
+  alias <%= inspect schema.module %>
+  alias <%= inspect scope.module %>
+
+  @doc """
+  Subscribes to scoped notifications about any <%= schema.singular %> changes.
+
+  The broadcasted messages match the pattern:
+
+    * {:created, %<%= inspect schema.alias %>{}}
+    * {:updated, %<%= inspect schema.alias %>{}}
+    * {:deleted, %<%= inspect schema.alias %>{}}
+
+  """
+  def subscribe_<%= schema.plural %>(%<%= inspect scope.alias %>{} = <%= scope.name %>_scope) do
+    key = <%= scope.name %>_scope.<%= Enum.join(scope.access_path, ".") %>
+
+    Phoenix.PubSub.subscribe(<%= inspect context.base_module %>.PubSub, "<%= scope.name %>:#{key}:<%= schema.plural %>")
```

Comment:
I was wondering if we should have this under the scope somehow. `subscribe(current_scope, topic)` and `broadcast(current_scope, topic)`. But this also means a basic scope will need additional functions in order to work with generators.

Reply:
I think it's fine to have those in the context. People can extract to their scope if they want.